cmd/run.go

```diff
@@ -909,6 +918,21 @@ func putAppCommandInMeta(runConfig standalone.RunConfig, runState *runExec.RunEx
 	return nil
 }
 
+// putRunFilePathInMeta puts the absolute path of run file in metadata so that it can be used by the CLI to stop all apps started by this run file.
+func putRunFilePathInMeta(runE *runExec.RunExec, runFilePath string) error {
```

Comment:
`putAppCommandInMeta` still calls `runE` as `runState`, can you please update that?